src/store/modules/sidenavBoxes.js

```diff
@@ -0,0 +1,44 @@
+import { withMeta, createMetaModule } from '@/store/helpers'
+
+function initialState () {
+  return {
+    mapCollapsed: false,
+    storeCollapsed: false,
+    storesCollapsed: false,
+    groupCollapsed: false,
+  }
+}
+
+export default {
+  namespaced: true,
+  modules: { meta: createMetaModule() },
```

Comment:
You don't need the meta module because you're not doing async actions like server requests here. Meta provides pending status and validation errors.